Assets/Scripts/ProjectileHazardScript.cs

```diff
@@ -15,25 +16,25 @@ public class ProjectileHazardScript : MonoBehaviour
     void Start()
     {
         isMoving = false;
+        playerRocket = GameObject.Find("Rocket");
     }
 
     // Update is called once per frame
-    void Update()
+    private void Update()
     {
         Collider[] hitColliders = Physics.OverlapSphere(transform.position,pullRadius);
         
         foreach (var currentCollider in hitColliders)
         {
-            if (!isMoving){
+            if (isMoving){
+                GetComponent<Rigidbody>().AddForce(initialRocketPosition * strengthOfAttraction * Time.deltaTime);                
+            } else
```

Comment:
So I'm not sure if this was unpurpose, or if you forgot the brackets in the else statement.

Technically this will work. You do not need brackets in a conditional if your intention is to execute the statement on the same or following line of the if/else. For example:
```
if (isTrue)
doThing();
else
doOtherThing();
```

or even,

```
if(isTrue) doThing();
else doOtherThing(); 
```

both are valid.

In the case you have here, the if statement following the else is treated as the next statement. So this is technically valid.

All that said, let's not do this here. This feature is really just meant for one liners, and even then it's usually not recommended, especially for beginners. The reason is two fold. For one thing, this trick can really easily lead to hard to debug syntax errors, and then also it takes away from the readability of the code.

Let's always wrap our conditionals in brackets. Preferably in this style:

```
if (isTrue){
    doSomething();
else{
    doSomethingElse();
}
```